tests/test_cell_statistics.py

```diff
@@ -68,20 +68,20 @@ def test_segmentation_widget_run_button(
     )
 
 
-def test_calculate_cell_statistics(
```

Comment:
This test loads a pre-calculated data file so think will fail when we change stuff